src/Components/Blueprints/helpers/onPremToHostedBlueprintMapper.tsx

```diff
@@ -234,8 +234,7 @@ export const mapHostedToOnPrem = (
         name: u.name,
         key: u.ssh_key || '',
         groups: u.groups || [],
-        password: '',
-        isAdministrator: u.groups?.includes('wheel') || false,
```

Comment:
Why got `isAdministrator` removed?

Reply:
`isAdministrator` isn't support in the cloudapi, it's just an image builder api thing.